tofu/geom/_core_optics.py

```diff
@@ -788,6 +788,14 @@ def plot_rockingcurve(self, lamb=None, n=None, sigma=None,
             ang_units=ang_units, axtit=axtit, color=color,
             fs=fs, ax=ax, legend=legend)
 
+    def compute_rockingcurve(
+        self, ih=None, ik=None, il=None, lamb=None,
+    ):
+        import tofu.spectro._rockingcurve as _rockingcurve
```

Comment:
Better practice: put the `import` statement at the top of the file

Reply:
Have been done.